src/category_theory/limits/types.lean

```diff
@@ -60,7 +60,7 @@ lemma is_limit_equiv_sections_symm_apply
   c.π.app j ((is_limit_equiv_sections t).symm x) = (x : Π j, F.obj j) j :=
 begin
   equiv_rw (is_limit_equiv_sections t).symm at x,
-  simp,
+  rw is_limit_equiv_sections_apply,
```

Comment:
Are these changes related to the tactic change?

Reply:
No these are random small optimizations (I believe rewrites are faster than simps). Let me know if I should drop these changes.

Reply:
If these are random small optimizations, these should be part of a separate `chore` PR I believe.